src/partials/footer.html

```diff
@@ -1,4 +1,25 @@
 <footer class="footer">
-  <p>© 2023 | All Rights Reserved | Developed with <3 by <a href="#">Harnaś Team</a></p>
+  <div class="footer__text">
+    <div class="footer__text-first">© 2023 | All Rights Reserved |</div>
```

Comment:
poprosiłbym małe zmiany jeśli chodzi o semantykę czyli nie wszystko w divy tylko tutaj np. dać link <a>

Reply:
dałam to w divy żebym miała dwa kontenery, które w wersji mobilnej mi się układają w kolumnie a w desktopie w jednej linii

Reply:
w <p> albo w <span> mi to się nie chce zrobić content center tylko spycha text do dolnej granicy, albo jestem zmęczona i coś mi sie myli, albo musi to tak zostać

Reply:
w p i spanie* (tam na początku miało być)

Reply:
to dodaj te zmiany i wyślij jeszcze raz brancha to wtedy przejdzie luzem